egs/callhome_diarization/v1/diarization/make_rttm.py

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Copyright  2016  David Snyder
+# Apache 2.0.
+# TODO, script needs some work, error handling, etc
+
+import sys
+
+segments_fi = open(sys.argv[1], 'r').readlines()
+label_fi = open(sys.argv[2], 'r').readlines()
+
+# File containing speaker labels per utt
+seg2label = {}
+for l in label_fi:
+  seg, label = l.rstrip().split()
+  seg2label[seg] = label
+
+# Segments file
+utt2seg = {}
+for l in segments_fi:
+  seg, utt, s, e = l.rstrip().split()
+  if utt in utt2seg:
+    utt2seg[utt] = utt2seg[utt] + " " + s + "," + e + "," + seg2label[seg]
+  else:
+    utt2seg[utt] = utt + " " + s + "," + e + "," + seg2label[seg]
+
+
+# TODO Cut up the segments so that they are contiguous
```

Comment:
There is script that does something very similar except for post-processing. Perhaps this can be used. https://github.com/kaldi-asr/kaldi/blob/master/egs/wsj/s5/steps/segmentation/convert_utt2spk_and_segments_to_rttm.py

Reply:
@mmaciej2, I think this script needs to go, and be replaced with the one @vimalmanohar mentioned.